letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
ah, this way. Well, the db filter *must* resolve to exactly one database anyways, because otherwise it's not defined in which data dir you end up. And we want a database specific data directory because that's the only place where it's safe to assume that we can write, and because we don't want different databases to overwrite each other's certificates/keys. Then I also understand you followup changes to remove `env` here, but I don't agree with them.